packages/libretro/dolphin/patches/dolphin-01-buildfix.patch

```diff
@@ -0,0 +1,30 @@
+diff --git a/Source/Core/DolphinLibretro/CMakeLists.txt b/Source/Core/DolphinLibretro/CMakeLists.txt
+index 7a09e340e3..c588fe8125 100644
+--- a/Source/Core/DolphinLibretro/CMakeLists.txt
++++ b/Source/Core/DolphinLibretro/CMakeLists.txt
+@@ -35,19 +35,10 @@ endif()
+ 
+ if(NOT MSVC AND NOT CLANG)
+    set(CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} -Wl,--no-undefined")
+-else()
+-   target_link_libraries(dolphin_libretro PRIVATE
+-      core
+-      uicommon
+-      ${LIBS}
+-   )
+ endif()
+    
+-if(LIBRETRO_STATIC)
```

Comment:
Would there be value in having `LIBRETRO_STATIC` defined by default for the core? Would save having to maintain the patch, and the if() statements around it.

Reply:
I believe specifically having `LIBRETRO_STATIC` defined wouldn't be correct, because that would change the values being passed into `target_link_libraries`, right? But I do like your suggestion of changing the defined values to match what we need to happen, instead of patching the cmake.

Please correct me if I'm wrong about the first thing, I wasn't the original author of the change, so I'm not totally sure if `-static-libgcc -static-libstdc++` is wanted there.

Reply:
@plaidman do you want to make more changes on this before merging the PR?

Reply:
updated to remove patch file. good suggestion @RobLoach